lib/scan/mail_id_harvest/engine.py

```diff
@@ -0,0 +1,117 @@
+import time
+import requests
+import socks
+import socket
+import json
+import re
+from bs4 import BeautifulSoup as BS
+from core._time import now
+from xml.etree import ElementTree as ET
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using ElementTree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace ElementTree with the equivalent defusedxml package, or make sure defusedxml.defuse_stdlib() is called.](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5096791)